### PR TITLE
328 branded panel

### PR DESCRIPTION
**Pull request by jrbarnes9**
Requires documentation

### CODE REVIEW

**Comment by jrbarnes9** [comment]
[Branded panel](https://trello.com/c/n7ei1C9Q/328-branded-panel)

**Comment by bameyrick** [comment]
Looks like commit 35618c8 shouldn't have been part of this? Otherwise looks good to me

**Comment by bameyrick** [comment]
Conflicts need resolving

File: src/components/pagination/index.njk [hunk 1]
```diff
@@ -13,13 +13,16 @@ Pagination is used to navigate a series of pages in a linear or semi-linear mann
     patternlibExample({"path": "components/pagination/examples/pagination/index.njk"})
 }}
 
+<<<<<<< HEAD
```
Comment (bameyrick) [inline src/components/pagination/index.njk]: Don't think you merged this file correctly

File: src/components/panel/examples/branded/index.njk [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+"theme": census
+---
+{% from "components/panel/_macro.njk" import onsPanel %}
+{{ 
+    onsPanel({
```
Comment (bameyrick) [inline src/components/panel/examples/branded/index.njk]: Might be better to use the {% call onsPanel %} method for this one, as your body has a fair bit of html in there.

File: src/views/layouts/_master.njk [hunk 3]
```diff
@@ -6,6 +6,15 @@
     {% set page = {} %}
 {% endif %}
 
+<<<<<<< HEAD
```
Comment (bameyrick) [inline src/views/layouts/_master.njk]: Another failed merge

File: src/scss/settings/_census.scss [hunk 4]
```diff
@@ -1,6 +1,8 @@
 // Census brand
-$color-branded: #93328e;
-$color-branded-tint: rgba(227, 77, 219, 0.15);
+$census-purple: #93328e;
```
Comment (bameyrick) [inline src/scss/settings/_census.scss]: You've merged this the wrong way around

File: src/components/panel/examples/branded/index.njk [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+"theme": census
+---
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% call onsPanel({
+        type: "branded"
+    }) 
+%}
+   This guidance is for the paper questionnaire.<br>View the <a href="#">question guidance for completing the census online.</a>
```
Comment (bameyrick) [inline src/components/panel/examples/branded/index.njk]: Probably still need  `<p>` tags here
```suggestion
   <p>This guidance is for the paper questionnaire.<br>View the <a href="#">question guidance for completing the census online.</a></p>
```

**Pull request merged**